crates/matrix-sdk-ui/src/timeline/inner/mod.rs

```diff
@@ -835,6 +835,15 @@ impl<P: RoomDataProvider> TimelineInner<P> {
     ) -> Option<(OwnedEventId, Receipt)> {
         self.state.read().await.latest_user_read_receipt(user_id, &self.room_data_provider).await
     }
+
+    /// Get the ID of the timeline event with the latest read receipt for the
+    /// given user.
+    pub(super) async fn latest_user_read_receipt_timeline_event_id(
```

Comment:
As suggested elsewhere, could we rename the whole family `latest_user_read_event_id`? (or depart from the current naming scheme and name this `latest_read_event_by_user`)